mailpoet/lib/Form/Block/Date.php

```diff
@@ -116,9 +116,19 @@ public function getDateFormats(): array {
   }
 
   public function getMonthNames(): array {
-    return [__('January', 'mailpoet'), __('February', 'mailpoet'), __('March', 'mailpoet'), __('April', 'mailpoet'),
-      __('May', 'mailpoet'), __('June', 'mailpoet'), __('July', 'mailpoet'), __('August', 'mailpoet'), __('September', 'mailpoet'),
-      __('October', 'mailpoet'), __('November', 'mailpoet'), __('December', 'mailpoet'),
+    return [
```

Comment:
should we better late escape in L162?

Reply:
I was thinking about it, but I had a better feeling about it here because it seemed to me better visible. But if you have an opposite opinion. I can change that.

Reply:
@websupporter I have looked at it again, and I think it should be here because it is a public method that is reused in other places.